### PR TITLE
Add Scala.js shields/badges as SVG images

### PR DESCRIPTION
**Pull request by ochrons**
that can be used to indicate compatibility with Scala.js

Add a shield generator onto the Scala libraries page.

### CODE REVIEW

File: _data/library/versions.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+["0.6.0","0.6.1","0.6.2","0.6.3","0.6.4","0.6.5","0.6.6"]
```
Comment (sjrd) [inline _data/library/versions.yml]: Can this be written like this instead?
```yml
- 0.6.0
- 0.6.1
- 0.6.2
- 0.6.3
- 0.6.4
- 0.6.5
- 0.6.6
```
I think it's the same YAML value in both cases, isn't it? That would give nicer diffs in subsequent additions.

**Comment by sjrd** [comment]
Could we select by default the latest version (available through `{{ site.versions.scalaJS }}`)?

**Comment by sjrd** [comment]
That's all.

**Comment by ochrons** [comment]
The latest version is automatically selected, as the version list is reversed in the selection box, so latest is always at the top.

**Comment by sjrd** [comment]
LGTM, OK.

**Pull request merged**